src/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -16,6 +16,7 @@ namespace System.ComponentModel
     /// </summary>
     public class EnumConverter : TypeConverter
     {
+        private static readonly char[] s_comma = {','};
```

Comment:
Consider renaming this to `s_separators` to convey the fact that this is a separator array.